### DISCUSSION TITLE
Publish build artifacts to GitHub releases?

### DISCUSSION THREAD

**Discussion by LukeCarrier** [gitbutlerapp/gitbutler#3987] [Q&A] [answered]
I suspect the most GitButler release at the time of writing (0.12.0) may have broken my ability to create commits. From a quick scan of recent commits it looks like this might be due to signing.

Since you're already tagging releases, is there any particular reason you don't upload the build artifacts for each release as release assets? This would have enabled me to quickly downgrade and go about my day and defer the troubleshooting 😅

**Comment by Byron** [comment]
I love the idea, personally I was also looking at GH releases when first coming here, thinking they would be the baseline.

While these are not available though, do you think you could try to *upgrade* to get a fix? Nightly releases can be found [here](https://app.gitbutler.com/downloads).

**Reply by LukeCarrier** [reply]
I can do one better: I built the code expecting to need to go on a debugging journey and found it already fixed. I don’t know as the nightly at the time would have included that fix, though 😅

**Reply by Byron** [reply]
It's good to be a dev :)!

If the commit with the fix was done yesterday, it's likely that the daily nightly includes it. In the worst case, tomorrow an official nightly build is available that definitely includes.

**Reply by Byron** [reply, marked as answer]
Oh, and here is the issue to follow for GitHub releases: https://github.com/gitbutlerapp/gitbutler/issues/3421 .
